.github/workflows/plugins-test.yaml

```diff
@@ -494,3 +494,34 @@ jobs:
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run mysql 5.1.2-8.0.15 (53)
         run: bash test/plugin/run.sh mysql-scenario
+
+  Oracle:
```

Comment:
Oracle just have one version to test, so you could add this to a faster group, rather than add a new one.

Reply:
ok